modules/contact-form/admin.php

```diff
@@ -863,6 +863,8 @@ function grunion_recheck_queue() {
 	foreach ( $approved_feedbacks as $feedback ) {
 		$meta = get_post_meta( $feedback->ID, '_feedback_akismet_values', true );
 
+		$meta['recheck_reason'] = 'recheck_queue';
```

Comment:
Two things:

1. By setting this, the `( ! $meta )` conditional below it would never match. We should add this after that `continue` unless we intend to change behavior.

2. If we intend to change the behavior, we need to make sure `$meta` is an array before adding to it. If, somehow, it's a string, it'll throw a warning or error on PHP 7.1

Reply:
Re: #1, you're absolutely right. I originally prepped this patch before the patch that added the `! $meta` check and forgot to take that into account.

For #2, is it possible for $meta to not be an array (if it's not falsy), outside of someone manually editing the DB?

Reply:
For item 2 is probably me being overly cautious. An unit test sets the value to `1` in one place, but it appears to always be an array. There's one instance of the same value that is eventually saved to the meta casted as an array when foreach'ing it.

I don't see any filters that would touch that meta before it is saved in the db, so probably fine to ignore me on that point.